code/procs/antagonist_procs.dm

```diff
@@ -76,29 +76,29 @@
 		R = traitor_mob.belt
 		loc = "on your belt"
 	if (!R && istype(traitor_mob.l_hand, /obj/item/storage))
-		var/obj/item/storage/S = traitor_mob.l_hand
-		var/list/L = S.get_contents()
-		for (var/obj/item/device/radio/foo in L)
+		var/list/cont = list()
+		SEND_SIGNAL(traitor_mob.l_hand, COMSIG_STORAGE_GET_CONTENTS, cont)
+		for (var/obj/item/device/radio/foo in cont)
```

Comment:
Alternatively I could do
```
var/list/cont = list()
SEND_SIGNAL(traitor_mob.l_hand, COMSIG_STORAGE_FIND_TYPE, /obj/item/device/radio, cont)
if (cont.len)
    R = cont[1]
    ...
```
But it feels weird to pass a list and just get the first item, though it's slightly more efficient. Thoughts?

Reply:
This doesn't get called enough for the performance to matter. I don't think there's that much use for the FIND_TYPE component as of right now.